_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,236 @@
+---
+title: 'Bitcoin Optech Newsletter #103'
+permalink: /en/newsletters/2020/06/24/
+name: 2020-06-24-newsletter
+slug: 2020-06-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a newly published fee ransom attack against
+LN users, links to continued discussion about the attack against LN
+atomicity, and shares a reminder about collision attacks on
+RIPEMD160-based addresses in multiparty protocols.  Also included are
+our regular sections with popular questions and answers from the Bitcoin
+StackExchange, a list of releases and release candidates published this
+week, and notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **LN fee ransom attack:** René Pickhardt [publicly
+  disclosed][pickhardt post] a vulnerability to the Lightning-Dev
+  mailing list that he had previously privately disclosed to LN
+  implementation maintainers almost a year ago.  In the current LN
+  protocol, the party who initiates opening a channel must commit to
+  paying any onchain transaction fees that will be required to later
+  close that channel.  The party paying the fees also gets to choose the
+  feerate used---but the security of both parties depends on choosing an
+  appropriate feerate, so the counterparty may close the channel if they
+  think the chosen feerate is too low.  To avoid such unnecessary
+  closes, [BOLT2][] recommends the fee-paying party commit to fees five
+  times higher than the minimum reasonable estimate---high enough to
+  ensure the counterparty will be satisfied even if they're using a
```

Comment:
From BOLT2: *Given the variance in fees, and the fact that the transaction may be spent in the future, it's a good idea for the fee payer to keep a good margin (say 5x the expected fee requirement); but, due to differing methods of fee estimation, an exact value is not specified.*

- "ensure" and "high enough": I wonder if these words are too reassuring for describing a hopeful, non-guaranteed margin of safety (IIUC)

- "recommends": the 5x looks like more of an off-the-cuff, to-give-an-idea-of-order-of-magnitude example than a recommendation